test/ectest.c

```diff
@@ -1084,6 +1086,86 @@ static int char2_field_tests(void)
     return r;
 }
 
+static int check_bn_mont_ctx(BN_MONT_CTX *mont, BIGNUM *mod, BN_CTX *ctx)
+{
+  int ret = 0;
+  BN_MONT_CTX *regenerated = BN_MONT_CTX_new();
```

Comment:
Look at other files to see how the tests are done.. using if(!TEST_XXX()) e.g.
if (!TEST_ptr(...........))
    goto err;